source/_components/climate.markdown

```diff
@@ -20,6 +20,39 @@ climate:
   platform: demo
 ```
 
+## {% linkable_title Properties %}
+
+Every climate device has a set of properties that control the operation of the climate device. These properties can be set via the frontend or by calling services and are reported in the attributes of the device's state. Note that not all platforms implement all the properties.
+
+We explain each of the properties and the possible choices for them shortly:
+
+* The **action** property specifies the overall operational mode that the device is currently in. Choices are:
```

Comment:
This is what in my opinion should have been the options for operation_mode. I would rather fix that instead of adding a new one.

Reply:
I'm not talking about adding. I'm talking about replacing.

Reply:
I think Operation expresses better than Action the types here. But that is my personal view and preference of course.